src/components/MajorFilter.jsx

```diff
@@ -0,0 +1,63 @@
+import React, { useState } from 'react';
+import { Button, Autocomplete, Popover, Chip, TextField } from '@mui/material';
+import { MAJORS } from '../types/options';
+import ArrowDropDownIcon from '@mui/icons-material/ArrowDropDown';
+
+function MajorFilter() {
+  const [anchorEl, setAnchorEl] = useState(null);
+  const [majors, setMajors] = useState([]);
+  const [isOpen, setIsOpen] = useState(false);
+
+  return (
+    <>
+      <Button
+        variant="outlined"
+        onClick={(e) => setAnchorEl(e.currentTarget)}
+        endIcon={<ArrowDropDownIcon color="primary" />}
+        sx={{ margin: 1, height: (theme) => theme.spacing(4) }}>
+        Majors
+      </Button>
+      <Popover
+        open={Boolean(anchorEl)}
+        anchorEl={anchorEl}
+        onClose={() => setAnchorEl(null)}
+        anchorOrigin={{ vertical: 'bottom', horizontal: 'left' }}
+        transformOrigin={{ vertical: 'top', horizontal: 'left' }}
+        sx={{ maxWidth: 400 }}>
+        {majors.map((major) => (
+          <Chip
```

Comment:
This might not be a problem when we implement it with the collapsable menu.

<img width="441" alt="Screen Shot 2022-01-04 at 2 14 54 PM" src="https://user-images.githubusercontent.com/40483569/148136224-897f40fa-eac3-48b5-b035-a12f6dd0103f.png">

Reply:
I think setting a limit is fine. Let's say maybe 5 or 10?